cmd/ef_tests/levm/runner/revm_runner.rs

```diff
@@ -420,8 +420,8 @@ pub fn _ensure_post_state_revm(
             match test.post.vector_post_value(vector).expect_exception {
                 // Execution result was successful but an exception was expected.
                 Some(expected_exception) => {
-                    let error_reason = format!("Expected exception: {expected_exception}");
-                    println!("Expected exception: {expected_exception}");
+                    let error_reason = format!("Expected exception: {expected_exception:?}");
+                    println!("Expected exception: {expected_exception:?}");
```

Comment:
Quick question, couldn't the `println` print the "error_reason" variable instead of the re-writing the string?

Reply:
Removed the print in [Remove exception print, already prints in report](https://github.com/lambdaclass/ethrex/pull/1459/commits/d3524d88f2cef2fb21cfdb89c716b59eb040bcaa)